ReSwiftTests/TypeHelperTests.swift

```diff
@@ -51,21 +51,18 @@ class TypeHelper: QuickSpec {
 
             it ("doesn't call if source type can't be casted to function signature type") {
                 var called = false
-                let reducerFunction: (AppState1?, Action) -> AppState1 = { state, action in
+                let reducerFunction: (Action, AppState1?) -> AppState1 = { action, state in
                     called = true
 
                     return state ?? AppState1()
                 }
 
-                withSpecificTypes(AppState2(), action: StandardAction(""),
+                withSpecificTypes(StandardAction(""), state: AppState2(),
                     function: reducerFunction)
 
                 expect(called).to(beFalse())
             }
-
         }
-
     }
-
 }
-// swiftlint:enable function_body_length
+// swiftlint:enable function_body_length
```

Comment:
Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)